src/robotide/ui/tagdialogs.py

```diff
@@ -121,10 +130,10 @@ def _execute(self):
         self.SortListItems(self.sort_state[0], self.sort_state[1])
 
     def update_footer(self):
-        footer_string = "Total tests %d, Tests with tags %d, Unique tags %d," \
-                        " Currently selected tests %d" % \
-            (self.total_test_cases, len(self.tagged_test_cases),
-             self.unique_tags, len(self.selected_tests))
+        footer_string = "Total tests %d, Tests with tags %d, Unique tags %d, \
+Currently selected tests %d" % (self.total_test_cases,
+                                len(self.tagged_test_cases), self.unique_tags,
+                                len(self.selected_tests))
```

Comment:
Is this just a formatting change? Should preferably be in a separate commit (or even pr). More importantly,  I don't think cutting a string like

```python
    "Example string \
here is."
```

is a good idea. I'd use either

```python
    "Example string " \
    "here is."
```

or probably

```python
    ("Example string "
     "here is.")
```

The latter is also what [PEP-8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) recommends:

> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.